packages/hashi/src/mainClient.js

```diff
@@ -104,6 +120,94 @@ export default class MainClient {
       });
     });
   }
+
+  __fetchContentData(message) {
+    // based on the incoming information, get data
+    // from Kolibri to pass back to the iframe
+
+    // if filtering by optional params
+    if (message.options) {
+      let getParams = {};
```

Comment:
Can use `const` for both `getParams` and `options` here as you are never reassigning their values. You've assigned it to an object, and even though you are changing the attributes of that object, you are not changing the referent of the variable.